### ISSUE TITLE
edit sponsors in carrer mode

### ISSUE THREAD

**Issue by grzegorzkutkowski** [xAranaktu/FIFA-23-Live-Editor#265]
Hi, its possible to edit sponsors on kits for created club in carrer mode? :) How to do this?

**Comment by theoriginalskengman** [comment]
You could head across to Fifa Kit Creator and make the kits yourself.

**Comment by grzegorzkutkowski** [comment]
> You could head across to Fifa Kit Creator and make the kits yourself.

Its easy? Any template oryginal suit and only logo change? Can you explain More ?  :)

**Comment by theoriginalskengman** [comment]
Have a look in YouTube for a channel called "Sparring DK". He was a series of in depth tutorial on how to create the kits and how to make the mod to inject into the game.

**Comment by grzegorzkutkowski** [comment]
> Have a look in YouTube for a channel called "Sparring DK". He was a series of in depth tutorial on how to create the kits and how to make the mod to inject into the game.

Ok , thanks. Its possivble to change kits for my created team (carrer mode?)

**Comment by theoriginalskengman** [comment]
Yes, I've done in for fifa 22 and 23. Again look through his series on kit creation. It goes in depth of how to add kits to teams, custom clubs, add extra kits to teams etc etc

**Comment by grzegorzkutkowski** [comment]
> Yes, I've done in for fifa 22 and 23. Again look through his series on kit creation. It goes in depth of how to add kits to teams, custom clubs, add extra kits to teams etc etc

Yes but no tutorial how to do this for created team. I create virtual team example: Created Team FC and i want change kits. How to do this for created team?

**Comment by theoriginalskengman** [comment]
My, have you ever tried Google? Literally first thing that comes up is Tut for it...
https://youtu.be/OYQZGZWLPjE


Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by theoriginalskengman** [comment]
Have you ever tried this thing called Google? I just searched for what you wanted and the first vid that come up was  TuT for exactly what you're asking about.....
